cmake/GenerateBuiltins.cmake

```diff
@@ -243,6 +241,9 @@ function(builtin_to_cpp bit os_name arch supported_archs supported_oses resultFi
         if (${os_name} STREQUAL "macos")
             # -isystem/iusers/MacOSX10.14.sdk.tar/MacOSX10.14.sdk/usr/include
             set(includePath -isystem${ISPC_MACOS_SDK_PATH}/usr/include)
+        elseif (${os_name} STREQUAL "windows")
+            # -isystem/xwin/crt/include -isystem/xwin/sdk/include/ucrt
+            set(includePath -isystem${ISPC_WINDOWS_SDK_PATH}/crt/include -isystem${ISPC_WINDOWS_SDK_PATH}/sdk/include/ucrt)
```

Comment:
@nurmukhametov Couple you please verify that "default" build without cross compilation enabled is not affected?